lms/static/scripts/ui-playground/components/ErrorComponents.js

```diff
@@ -100,221 +101,115 @@ function OAuth2RedirectErrorAppExample({ errorCode = '' }) {
-        <p>
-          These errors arise from our own proxy API, and also from integrations
-          with other APIs (e.g. Google Drive).
-        </p>
-        <p>TODO</p>
       </div>
 
       <Library.Pattern title="ErrorDisplay">
         <Library.Example>
           <p>
-            All of the errors described here, with rare exception, use the{' '}
-            <code>ErrorDisplay</code> component to show their core details.
-          </p>
-          <p>
-            It is intended to be used within a Modal context, and provides a{' '}
-            <code>Scrollbox</code> to scroll content if it is too tall for the
-            containing element.
+            The <code>ErrorDisplay</code> component is used to show information
+            about errors to users. It is intended to be used within a Modal
+            context, and provides a <code>Scrollbox</code> to scroll content if
+            it is too tall for the containing element.
           </p>
           <p>
-            When information about an error is displayed to a user, we show:
+            When information about an error is displayed to a user via{' '}
+            <code>ErrorDisplay</code>, we show:
           </p>
           <ul>
             <li>
               <p>
-                <b>Prepared/canned text</b> based on <b>error code</b>
-              </p>
-              <p>
-                Canned text for different kinds of errors is defined in the
-                front-end app, for example, instructions on how to rectify a
-                missing Canvas file. Which canned text gets displayed is
-                determined by an error code provided by the back end. Depending
-                on the error context, this error code may either be in the JS
-                configuration object (error {'"modes"'}) or within the response
-                body of a problematic request to our proxy API.
-              </p>
-              <p>
-                Canned text is <b>sometimes</b> shown. It depends on whether a
-                recognized error code is provided. It is not relevant to all
-                error cases.
+                Any available <b>prepared/canned text</b> based on{' '}
+                <b>error code</b>. This is optionally provided by the front-end
+                app as <code>children</code> to the component.
               </p>
             </li>
             <li>
               <p>
-                A brief <b>contextual hint</b>
-              </p>
-              <p>
-                Also called a <b>description</b>, this text is provided in some
-                cases by the front-end app to provide context for the error,
-                e.g. {'"There was an error fetching book chapters"'}.
-              </p>
-              <p>
-                Contextual hints are <b>sometimes shown</b>.
+                A brief, optional <b>contextual hint</b>, as the{' '}
+                <code>description</code> prop. This comes from the front-end
```

Comment:
Why not refer to this directly as the `description` prop?

Reply:
On the demos, I would refer to:
* `With description and error.message`
* `Without description but with error.message`
* `Without description or error.message`
* `Without description, error.message or children`

Reply:
Great feedback. This _was_ quite confusing. I renamed these examples...